src/components/spaces/organisms/organisms-container.tsx

```diff
@@ -90,7 +90,7 @@ export class OrganismsContainer extends BaseComponent<IProps, IState> {
             {
               organismsMouse != null &&
                 <OrganelleWrapper zoomLevel={zoomLevel} elementName={`organelle-wrapper-${rowIndex}`}
-                  rowIndex={rowIndex} width={width}/>
+                  rowIndex={rowIndex} width={width} mode={mode}/>
```

Comment:
Still not entirely sure why these props need to be passed explicitly like this, but it did the trick!

Reply:
Again, there may be something we're not doing right with MST. OrganelleWrapper ought to be able to just bring these things in directly from the model observers, right? But somehow the fact that it's in an array or something means that it doesn't trigger a notification when things change.